rosbag2_storage_mcap/src/message_definition_cache.cpp

```diff
@@ -63,18 +68,22 @@ static std::set<std::string> parse_msg_dependencies(const std::string& text,
   return dependencies;
 }
 
-static std::set<std::string> parse_idl_dependencies(const std::string& text) {
+static std::set<std::string> parse_idl_dependencies(const std::string & text)
+{
   std::set<std::string> dependencies;
 
   for (std::sregex_iterator iter(text.begin(), text.end(), IDL_FIELD_TYPE_REGEX);
-       iter != std::sregex_iterator(); ++iter) {
+    iter != std::sregex_iterator(); ++iter)
+  {
     dependencies.insert((*iter)[1]);
   }
   return dependencies;
 }
 
-std::set<std::string> parse_dependencies(Format format, const std::string& text,
-                                         const std::string& package_context) {
+std::set<std::string> parse_dependencies(
+  Format format, const std::string & text,
+  const std::string & package_context)
+{
```

Comment:
This is a good example why I prefer clang-format instead of the ~uglify~, sorry uncrustify